GiTinder/Models/User.cs

```diff
@@ -24,6 +25,10 @@ public class User
         [JsonProperty("avatar_url")]
         public string Avatar { get; set; }
         public string Repos { get; set; }
+        [NotMapped]
+        public List<string> RawCodeFilesUrls { get; set; }
+        [JsonProperty("raw_code_urls")]
+        public string FiveRawCodeFilesUrls { get; set; }
```

Comment:
@Jonathan-Bonnin this should be included in the `ProfileResponse` model as well as `snippets`

Reply:
done, although obviously not visible here :)